docs/release-notes/2.38.0.md

```diff
@@ -0,0 +1,231 @@
+
+
+For example:
+
+    ```sh
+    SELECT * /* cluster=’east’ */ FROM city;   
+    ```
+You can enable this feature as follows:
+
+CLI - Pass the parameter `comments-parsing`
+UI - It is enabled by default. To disable, uncheck the **Disable comments parsing** checkbox in the **Additional Options** section. This is supported for for PostgreSQL and MySQL.
+
+![!image](../_images/PMM_QAN_Parsing.png)
+
+
+### New PMM Agent Down alert template
+
+We've added a new default alert template to  [Percona Alerting](../get-started/alerting.md). 
+
+The new **PMM Agent down** alert template monitors your Node status to notify if the Agent is down. Before using this template:
+
+ - ensure the PMM Agent is healthy and present on the Database Node.
+-  check that PMM Client is on version 2.38 version or later.
+
+![!image](../_images/Agent_down.png)
+
+## New Features
+
+- [PMM-2852](https://jira.percona.com/browse/PMM-2852) -   [QAN]: Starting with PMM 2.38.0, MySQL and PostgreSQL supports collection of labels from the comments placed in the SQL queries.
+- [PMM-11961](https://jira.percona.com/browse/PMM-11961) - [Inventory]: Starting with PMM 2.38.0, we have enhanced the **Inventory** page by improving the **Nodes** tab. With this change, you can get more context on your inventory objects, such as node, and more actionable information for exploring and resolving possible issues.
+- [PMM-11962](https://jira.percona.com/browse/PMM-11962) - [Inventory]: Starting with PMM 2.38.0, you can filter the nodes and services on the **Inventory** page using the parameters such as ID, names, status, etc., instead of scrolling through an exhaustive list of nodes or services.
+- [PMM-12087](https://jira.percona.com/browse/PMM-12087) - [Inventory]: Added tooltips to explain the meaning of the different services statuses in the Inventory view.   
+
+
+## Improvements
+
+- [PMM-9367](https://jira.percona.com/browse/PMM-9367) -   [Backups]: MongoDB Backup: Sharded Cluster Backup
+- [PMM-11250](https://jira.percona.com/browse/PMM-11250) - [Backup]: Allow user to specify target directory for backup artifact.
+- [PMM-12038](https://jira.percona.com/browse/PMM-12038) - [Backup]: Extended support for MongoDB backups to cover all authentication mechanisms available for MongoDB (including LDAP, x509 certs).
+- [PMM-12114](https://jira.percona.com/browse/PMM-12114) - Grafana has been upgraded to 9.2.18 due to CVE.
+- [PMM-12136](https://jira.percona.com/browse/PMM-12136) - We have migrated our base operating system on which we build our base image from CentOS 7 to Oracle Linux 9 to fix the CVE's.
+
+## Bugs Fixed
+
+- [PMM-11578](https://jira.percona.com/browse/PMM-11578) - [Alerting]: The **Last notified** column  under **Alerting > Fired Alerts** has been renamed to **Last triggered**. This better illustrates the fact that triggered alerts may not always generate a notification. This can happen when the contact points are not correctly configured.
```

Comment:
https://github.com/percona/pmm-doc/pull/1085#discussion_r1239396565
not resolved.

Reply:
@catalinaadam